packages/core/feature-flags/README.mdx

```diff
@@ -20,56 +20,53 @@ One example of invalid use cases are settings used during the `setup` lifecycle
 
-```typescript
-// <plugin>/server/index.ts
-import type { FeatureFlagDefinitions } from '@kbn/core-feature-flags-server';
-import type { PluginInitializerContext } from '@kbn/core-plugins-server';
-
-export const featureFlags: FeatureFlagDefinitions = [
-  {
-    key: 'myPlugin.myCoolFeature',
-    name: 'My cool feature',
-    description: 'Enables the cool feature to auto-hide the navigation bar',
-    tags: ['my-plugin', 'my-service', 'ui'],
-    variationType: 'boolean',
-    variations: [
-      {
-        name: 'On',
-        description: 'Auto-hides the bar',
-        value: true,
-      },
-      {
-        name: 'Off',
-        description: 'Static always-on',
-        value: false,
-      },
-    ],
-  },
-  {...},
-];
-
-export async function plugin(initializerContext: PluginInitializerContext) {
-  const { FeatureFlagsExamplePlugin } = await import('./plugin');
-  return new FeatureFlagsExamplePlugin(initializerContext);
-}
-```
+### Variations
+
+All the potential values that a feature flag can return based on the [evaluation rules](#evaluationsegmentation-rules). A feature flag should define at least 2 variations: the ON and the OFF states. The typical use case sets the OFF state to match the `fallback` value provided to [the evaluation APIs](https://docs.elastic.dev/kibana-dev-docs/tutorials/feature-flags-service#evaluating-feature-flags), although it's not a hard requirement and Kibana contributors might require special configurations.
+
+### Evaluation/Segmentation rules
+
+Set of rules used to evaluate the variation to resolve, based on the [evaluation context](#evaluation-context) provided by the app.
+
+The rules can vary from a percentage rollout per evaluation context, or more complex IF...THEN filters and clauses.
+
+Refer to the [Evaluation Context guide's examples](./evaluation-context.md#examples) for some typical scenarios.
+
+### Evaluation Context
+
+Kibana reports a set of properties specific to each ECH deployment/Serverless project to help define the [segmentation rules](#evaluationsegmentation-rules). A list of the currently reported context properties can be found in the [Evaluation Context guide](https://github.com/elastic/kibana-feature-flags/blob/main/docs/evaluation-context.md).
+
+### Feature Flag Code References
+
+In the Kibana repo we run a [GH Action](https://github.com/elastic/kibana/actions/workflows/launchdarkly-code-references.yml) that links existing Feature Flags to their code references. This helps us figure out which flags have been removed from the code and, which ones are still being used.
+
+> :information_source: New flags might take a few moments to be updated with their code references.
+> The job runs on every commit to the `main` branch of the [Kibana repository](https://github.com/elastic/kibana), so the wait can take from a few minutes to a few hours, depending on the Kibana repo's activity.
+
```

Comment:
Q: Do we need to share this with contributors if they won't have access to LaunchDarkly's UI (at least for now)?

Reply:
I figured it won't hurt and might help troubleshooting... If you think it will be confusing, I can remove